src/app/lib/config/services/portuguese.js

```diff
@@ -72,16 +72,16 @@ export const service = {
     showRelatedTopics: true,
     podcastPromo: {
       title: 'Podcast',
-      brandTitle: 'Que História!',
+      brandTitle: 'BBC Lê',
       brandDescription:
-        'Incríveis histórias reais que nem a mente mais criativa poderia ter inventado',
+        'A BBC lê para você algumas de suas melhores reportagens',
       image: {
-        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p09dsdxd.jpg',
-        alt: 'Que História!',
+        src: 'https://ichef.bbc.co.uk/images/ic/448xn/p09qw181.jpg',
```

Comment:
Not really sure what the difference is between using 'bbc' and 'bbci' in links.

Reply:
```suggestion
        src: 'https://ichef.bbc.co.uk/images/ic/$recipe/p09qw181.jpg',
```

the `$recipe` part will get replaced with the appropriate image size in the srcset attribute here https://github.com/bbc/simorgh/blob/latest/src/app/containers/PodcastPromo/index.jsx#L18-L21

This enables us to serve different sized images to different devices.